File: PYTHON/Cap05/c05ex05.py
# c05ex05.py

i = 1
while True:
  n = int(input("Entre um valor numerico: "))

  r = n * 3

  print("Resultado =", r)
  print()

  i += 1
  if not (i > 5):
    continue
  else:
    break

input("\nTecle <Enter> para encerrar... ")

File: SCALA/Cap05/c05ex05.scala
// c05ex05.scala

import scala.io.StdIn._

object Main {
  def main(args: Array[String]): Unit = {
    var I: Int = 1
    var N, R: Int = 0

    do {
      print("Entre um valor numerico: ")
      N = readInt()

      R = N * 3

      println("Resultado = " + R + "\n")

      I += 1
    } while (!(I > 5))

    print("\nTecle <Enter> para encerrar... ")
    readLine()
  }
}
